src/selectors/run-command.js

```diff
@@ -0,0 +1,20 @@
+import { createSelector } from 'reselect';
+
+const getSlicedPipeline = (state) => state.slice;
+const getNodesNames = (state) => state.node.fullName;
```

Comment:
The naming of our redux store needs to be revisited. `state.node.fullName` sounds like a string but once I see the structure, it is a dictionary of node ids and names. Not related to this PR, but I would like us to work on this in future @Huongg